test/end-to-end/core.sh

```diff
@@ -129,7 +129,7 @@ os::cmd::expect_success 'oc env -n default dc/docker-registry REGISTRY_MIDDLEWAR
 os::cmd::expect_success 'oc rollout status dc/docker-registry'
 os::log::info "Restore configured to enable mirroring"
 
-registry_pod="$(oc get pod -n default -l deploymentconfig=docker-registry --show-all=false --template='{{(index .items 0).metadata.name}}')"
+registry_pod="$(oc get pod -n default -l deploymentconfig=docker-registry --template '{{range .items}}{{if not .metadata.deletionTimestamp}}{{.metadata.name}}{{end}}{{end}}')"
```

Comment:
Don't know yet. It's just taking ~10 seconds on my laptop for the old pod to go from Terminating to 100% deleted.